backend/src/v2/cmd/driver/main.go

```diff
@@ -100,6 +110,15 @@ func validate() error {
 	if *driverType == "" {
 		return fmt.Errorf("argument --%s must be specified", driverTypeArg)
 	}
+	if *httpProxy == unsetProxyArgValue {
+		return fmt.Errorf("argument --%s must be specified", httpProxyArg)
+	}
+	if *httpsProxy == unsetProxyArgValue {
+		return fmt.Errorf("argument --%s must be specified", httpsProxyArg)
+	}
+	if *noProxy == unsetProxyArgValue {
+		return fmt.Errorf("argument --%s must be specified", noProxyArg)
+	}
```

Comment:
The error message isn't particularly clear that you can set these to empty values. How about this?
```suggestion
	if *httpProxy == unsetProxyArgValue {
		return fmt.Errorf("argument --%s is required but can be an empty value", httpProxyArg)
	}
	if *httpsProxy == unsetProxyArgValue {
		return fmt.Errorf("argument --%s is required but can be an empty value", httpsProxyArg)
	}
	if *noProxy == unsetProxyArgValue {
		return fmt.Errorf("argument --%s is required but can be an empty value", noProxyArg)
	}
```